SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialRPCService.cpp

```diff
@@ -2,33 +2,47 @@
 
 #include "Interop/SpatialRPCService.h"
 
+#include "WorkerSDK/improbable/c_trace.h"
 #include "Interop/SpatialStaticComponentView.h"
 #include "Schema/ClientEndpoint.h"
 #include "Schema/MulticastRPCs.h"
 #include "Schema/ServerEndpoint.h"
 #include "Utils/SpatialLatencyTracer.h"
+#include "EngineClasses/SpatialNetDriver.h" // For EventTracing
```

Comment:
If you need to `#include` a broad-sounding header file for a specific purpose ("For EventTracing") then might be worth splitting stuff out into more granular header files?